go.mod

```diff
@@ -74,7 +74,7 @@ require (
 	k8s.io/apimachinery v0.29.1
 	k8s.io/client-go v0.29.1
 	k8s.io/component-base v0.29.1
-	k8s.io/cri-api v0.29.1
+	k8s.io/cri-api v0.30.0-alpha.2.0.20240216190946-4e003cc3b0a4
```

Comment:
are we OK having the containerd 2.0 release with 1.30 pre-release CRI API? Kubernetes 1.30 release is set for mid-April, so unless 2.0 is later than that, we would be importing non-released API?

Reply:
I assume this is safe, as RRO is still alpha, and GA features are not affected by this pre-release CRI API.